docs/requirements/software-requirements.md

```diff
@@ -0,0 +1,234 @@
+# CLARIAH Technical Requirements for Software and Services
+
+## Introduction
+
+This document intends to specify minimal technical requirements for CLARIAH
+software and software services. It is aimed at developers and external
+evaluators. It provides a minimal list of specific and directly actionable criteria to hold the software
+against. The aim is to ensure a certain level of software quality and
+sustainability, and to lay out basic requiments needed for interoperability
+within the larger CLARIAH infrastructure.
+
+The document is loosely derived from [earlier work to establish Software Quality
+Guidelines](https://github.com/CLARIAH/software-quality-guidelines/) within CLARIAH-CORE, but attempts to condense this
+elaborate work to the most relevant and directly actionable points.
+
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”,
+“SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be
+interpreted as described in [RFC
+2119](https://datatracker.ietf.org/doc/html/rfc2119).
+
+## Basic Software Requirements
+
+### 1. The software's source code *MUST* be stored in a public **version control system** (VCS).
+
+1. The use of a publicly accessible VCS platforms such as Github, Gitlab, Bitbucket or SourceHut is *REQUIRED*. This may be self-hosted by an institution rather than outsourcing it to a third party.
```

Comment:
Just examples, so mention the two most common ones only:

```suggestion
1. The use of a publicly accessible VCS platforms such as GitHub or GitLab is *REQUIRED*. This may be self-hosted by an institution rather than outsourcing it to a third party.
```

Reply:
You're probably right.. it's just that I like SourceHut a lot, despite it being smaller :) It also shows that we don't strictly need to adhere to the exact same paradigms GitHub/gitlab/bitbucket use (SourceHut is more e-mail based). So I'll keep it as-is for now.